src/seashore/tests/test_shell.py

```diff
@@ -114,6 +115,18 @@ def test_repr(self):
         self.assertIn('myout', stringified)
         self.assertIn('myerr', stringified)
 
+    def test_trace(self):
+        """traceback has args"""
+        try:
+            raise shell.ProcessError(13, "myout", "myerr")
+        except shell.ProcessError:
+            trace = traceback.format_exc()
```

Comment:
W:125,12: Redundant use of assertTrue with constant value False (redundant-unittest-assert)